lib/v2/hunau/utils/NewsContent.js

```diff
@@ -0,0 +1,47 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+const cheerio = require('cheerio');
+
+async function NewsContent(link, department = '') {
+    try {
+        // 异步请求文章
+        const { data: response } = await got.get(link, {
```

Comment:
https://github.com/DIYgod/RSSHub/blob/9bdf0364f36b3afaf1004488958c8ec420c82d8a/lib/v2/hunau/utils/Common.js#L18
```suggestion
        const { data: response } = await got(link, {
```
Keep your code consistent.
docs: https://docs.rsshub.app/joinus/script-standard.html